resources/styles.css

```diff
@@ -211,3 +219,50 @@ input.tab-text {
     width: 20px;
     height: 20px;
 }
+
+
+
+
+html, body, .r_flex_container, #app{
```

Comment:
using html and body is not recommended, do we really need that?

Reply:
Actually not. I saw this on an example and just do it but I'm sure we could have the same effect in other way... I'm going to try to change it

Reply:
well... I'm not sure why but I can get the same effect without this XD 
I think what is applied to html and body:
  height: 100%;
    display: flex;
    flex-direction: column;
    margin: 0;

is inoffensive and (from now) necessary :/